tests/test-buttercup.el

```diff
@@ -15,6 +15,9 @@
 ;; You should have received a copy of the GNU General Public License
 ;; along with this program. If not, see <http://www.gnu.org/licenses/>.
 
+;;; Commentary:
+;;
+
```

Comment:
Adding an empty commentary because some linter requires it does not improve readability, either.